recipe/meta.yaml

```diff
@@ -76,20 +76,24 @@ outputs:
         - pickle5  # [py<38]
       run:
         - python
+        - aiosignal
         - attrs
         - click >=7.0
         - colorama
         - dataclasses  # [py<37]
         - filelock
+        - frozenlist
         - grpcio >=1.28.1, <=1.43.0
         - msgpack-python >=1.0.0, <2.0.0
         - numpy >=1.20
         - pickle5  # [py<38]
         - protobuf >=3.15.3
         - psutil
         - pyyaml
-        - redis-py >=3.5.0
+        - redis-py >=3.5.0  # [win]
```

Comment:
The `redis` dependency was still in 1.11.1:

https://github.com/ray-project/ray/blob/releases/1.11.1/python/setup.py#L272

But it's not included in 1.12.0:

https://github.com/ray-project/ray/blob/releases/1.12.0/python/setup.py#L266

However, in windows, we patch Redis:

https://github.com/ray-project/ray/blob/releases/1.12.0/python/ray/__init__.py#L77

I'm not completely familiar with these changes on the Ray core side. The build completed for linux without redis, but not for mac (as the `import ray` failed because the patch couldn't be applied). It seems to me it might be still required for windows, at least in that release.

Reply:
I can ask people on the core side to confirm

Reply:
https://github.com/ray-project/ray/pull/22979 this is the PR that removed the dependency

Reply:
It has been removed on master: https://github.com/ray-project/ray/commit/2a5c40a149cfae1e2254b531fa2da207a719e451

So I think we should just keep it for this release.